source/ShipInfoDisplay.cpp

```diff
@@ -143,6 +143,13 @@ void ShipInfoDisplay::UpdateAttributes(const Ship &ship, const PlayerInfo &playe
 
 	attributeHeaderLabels.push_back("model:");
 	attributeHeaderValues.push_back(ship.DisplayModelName());
+
+	if(scrollingPanel)
```

Comment:
why only there?
->comment

Reply:
It would exceed the ship info panel height I believe